object_manager_ns-record.go

```diff
@@ -0,0 +1,23 @@
+package ibclient
+
+func (objMgr *ObjectManager) CreateNSRecord(dnsview string, recordname string, ns string, addr []string) (*RecordNS, error) {
+
+	var addresses []ZoneNameServer
+	for _, addr := range addr {
+		addresses = append(addresses, ZoneNameServer{Address: addr})
+	}
```

Comment:
Hi @k0da , Can you please add some validations on errors that might rise? 
Ex: if recordname == "" {
      return nil, fmt.Errorf("recordname must not be empty")
}